### DISCUSSION TITLE
Unable to see DM feature option after installing the aem-core-wcm-component 13.2 package

### DISCUSSION THREAD

**Discussion by pkhand0** [adobe/aem-core-wcm-components#1428] [Q&A] [answered]
Hi Team,

For DM feature functionality ,i  installed package core.wcm.components.all-2.13.2.zip but still not seeing the Enable DM feature option

**Comment by pkhand0** [comment]
<img width="807" alt="Screen Shot 2021-03-04 at 6 53 19 PM" src="https://user-images.githubusercontent.com/28389958/109970332-ef291200-7d1a-11eb-860b-95b0bdb71c71.png">
![Uploading Screen Shot 2021-03-04 at 6.55.09 PM.png…]()

**Comment by pkhand0** [comment]
![Uploading Screen Shot 2021-03-04 at 6.55.09 PM.png…]()

**Comment by msagolj** [comment, marked as answer]
The option only becomes available in the policy dialog when Dynamic Media Cloud Service has be been configured on the instance:

https://github.com/adobe/aem-core-wcm-components/blob/master/content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/_cq_design_dialog/.content.xml#L45-L56

To set up the service see:
https://experienceleague.adobe.com/docs/experience-manager-cloud-service/assets/dynamicmedia/config-dm.html?lang=en#architecture-diagram-of-dynamic-media

**Reply by pkhand0** [reply]
Thanks alot. It worked.
